### PR TITLE
address chef_gem warnings

### PR DESCRIPTION
**Pull request by blairham**
This addresses the chef_gem warnings based off this blog post: http://jtimberman.housepub.org/blog/2015/03/20/chef-gem-compile-time-compatibility

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by blairham** [comment]
Looks like someone needs to bump Curry again, I have signed the SLA

File: recipes/default.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@
 # gems with precompiled binaries
 %w{ win32-api win32-service }.each do |win_gem|
   chef_gem win_gem do
+  	compile_time true if Chef::Resource::ChefGem.instance_methods(false).include?(:compile_time)
```
Comment (lamont-granquist) [inline recipes/default.rb]: you should fix whitespace here and below

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Pull request closed without merging**